generate_all_selectors.go

```diff
@@ -0,0 +1,230 @@
+//go:build ignore
+
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"gopkg.in/yaml.v3"
+)
+
+const outputFilename = "all_selectors.yml"
+
+type ChainDetails struct {
+	ChainSelector uint64 `yaml:"selector"`
+	ChainName     string `yaml:"name"`
+}
+
+// Structure for reading EVM chains
+type evmYamlData struct {
+	Selectors map[uint64]ChainDetails `yaml:"selectors"`
+}
+
+// Final output structure
+type allSelectorsData struct {
```

Comment:
We have similar data type in extra_selectors go as well https://github.com/smartcontractkit/chain-selectors/blob/main/extra_selectors.go#L10-L18
Can we pull these and the parseYml logic etc into some other lib to reuse?